lib/kredis/connections.rb

```diff
@@ -3,11 +3,12 @@
 module Kredis::Connections
   mattr_accessor :connections, default: Hash.new
   mattr_accessor :configurator
+  mattr_accessor :redis_connector, default: ->(config){ Redis.new(config) }
```

Comment:
Would just go with `connector` – it's all redis here!

Reply:
👍 `connector` it is, thanks!